.gitignore

```diff
@@ -80,9 +80,12 @@ src/*.log
 tests/*.log
 
 # Unit test / coverage reports
+.cache/
 .coverage
 .tox
 .coverage.*
+__pycache__/
+*/__pycache__/
```

Comment:
Is git attempting to include these `__pycache__` directories now? I haven't seen that yet...